src/binaryen-c.h

```diff
@@ -64,6 +64,16 @@
 #define BINARYEN_API
 #endif
 
+#ifdef __cplusplus
+#define BINARYEN_REF(NAME)                                                     \
```

Comment:
@kripken You added your review before I added this macro, so wanted to double-check if this is okay or, if not, can you suggest a better name?

Reply:
I don't have a better idea, lgtm.